samples/multi-project/tye.yaml

```diff
@@ -16,4 +16,5 @@ services:
 - name: rabbit
   image: rabbitmq:3-management
   bindings:
-    - port: 5672
+    - protocol: amqp
```

Comment:
This isn’t required right?

Reply:
It is if you want to use `GetServiceUri` for rabbit. Per our discussion:

- We treat everything as URI-like by default
- We use connection strings only when you provided one

The URL that would be returned by `GetServiceUri` would then be `http://localhost:5672` which is not what they accept.


So we could either:

- Show this with a connection string in our examples
- Specify the protocol
- Introduce other methods `GetServiceHost` and `GetServicePort`. 

Obviously I prefer 2, or I would have done one of those other things. Your thoughts?

Reply:
Option 4, default to TCP and the caller would have to know they have to change it to amqp. Either way it seems like its not required but nice to do it here so that the APIs work.

Reply:
It's still important to try and agree on what we're showing. If we do what you're suggesting then when we prompt users at the commandline to get a URI during deployment, they will put in *something* 🤷‍♂. I think it's better in that regard to show that you have control over the URI - which is why I ended up here.